### PR TITLE
Minor docs improvements

### PR DESCRIPTION
**Pull request by abey79**
### What

Minor improvement based on comment on PRs I've yolo-merged earlier.

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] ~~I've included a screenshot or gif (if applicable)~~
* [x] ~~I have tested [demo.rerun.io](https://demo.rerun.io/pr/3654) (if applicable)~~

- [PR Build Summary](https://build.rerun.io/pr/3654)
- [Docs preview](https://rerun.io/preview/0c119d7b05cb1cfa3221838bd7891d70859a48e7/docs) 
- [Examples preview](https://rerun.io/preview/0c119d7b05cb1cfa3221838bd7891d70859a48e7/examples) 
- [Recent benchmark results](https://ref.rerun.io/dev/bench/)
- [Wasm size tracking](https://ref.rerun.io/dev/sizes/)

### CODE REVIEW

File: docs/content/concepts/annotation-context.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ By default, Rerun will automatically assign colors to each class id, but by defi
 you can explicitly determine the color of each class.
 
 * Python: [`rr.SegmentationImage`](https://ref.rerun.io/docs/python/HEAD/common/images/#rerun.SegmentationImag)
-* Rust: Log a [`Tensor`](https://docs.rs/rerun/latest/rerun/components/struct.Tensor.html) with [`TensorDataMeaning::ClassId`](https://docs.rs/rerun/latest/rerun/components/enum.TensorDataMeaning.html#variant.ClassId)
+* Rust: Log a [`rerun::SegmentationImage`](https://docs.rs/rerun/0.9.0-alpha.6/rerun/archetypes/struct.SegmentationImage.html)
```
Comment (emilk) [inline docs/content/concepts/annotation-context.md:75]: We often use 🐍 and 🦀 to prefix our Python and Rust links… though I don't know what we will use for C++ 😬

File: docs/content/concepts/entity-path.md [hunk 2]
```diff
@@ -27,7 +29,8 @@ However, it is also acceptable to leave implicitly "empty" elements in your path
 rr.log("camera/image", rr.Image(img))
 rr.log("camera/image/detections/points", rr.Points2D(points))
 ```
-Nothing needs to be explicitly logged to `"camera"` or `"camera/image/detection"` to make the above valid.
+Nothing needs to be explicitly logged to `"camera"` or `"camera/image/detection"` to make the above valid. The examples
+above are using the Python SDK, the
```
Comment (emilk) [inline docs/content/concepts/entity-path.md:33]: sentence fragment

File: docs/content/concepts/annotation-context.md (hunk 1 above)
Comment (abey79) [reply docs/content/concepts/annotation-context.md:75]: 🌊 i believe

File: docs/content/concepts/entity-path.md (hunk 2 above)
Comment (abey79) [reply docs/content/concepts/entity-path.md:33]: 🤦🏻

File: docs/content/concepts/annotation-context.md (hunk 1 above)
Comment (Wumpf) [reply docs/content/concepts/annotation-context.md:75]: did we definitely decide whether it's not 🦭

Comment (abey79) [reply docs/content/concepts/annotation-context.md:75]: that question is not for 0.9 to settle :)

**Review by emilk — APPROVED** [review]

**Pull request merged**